job-service-acceptance-tests/src/test/java/com/hpe/caf/jobservice/acceptance/JobServiceEndToEndIT.java

```diff
@@ -680,6 +680,7 @@ public void testCreateJobNoDelayNoPreReq() throws Exception {
         //retrieve job using web method
         final Job retrievedJob = jobsApi.getJob(partitionId, jobId, jobCorrelationId);
         LOG.info("--testCreateJobNoDelayNoPreReq job {} in partition: {}", retrievedJob.getId(), partitionId);
+        Thread.sleep(100); // Allows the scheduler to perform
```

Comment:
You could consider removing this test altogether.  It's trying to check an internal implementation detail so it's not a good end-to-end test.

Reply:
Removed test